packages/headless/src/controllers/knowledge/generated-answer/headless-answerapi-generated-answer.ts

```diff
@@ -160,10 +160,17 @@ export function buildAnswerApiGeneratedAnswer(
 
   subscribeToSearchRequest(engine as SearchEngine<StreamAnswerAPIState>);
 
+  const getCurrentAnswerState = () => {
+    const currentState = getState();
+    const params = selectAnswerApiQueryParams(currentState);
+    return selectAnswer(params)(currentState).data;
+  };
```

Comment:
Yea I think before merging this we should be more confident in the component.

I think I would hold off on changing this for now.